src/standalone/Vegetation/canopy_boundary_fluxes.jl

```diff
@@ -184,29 +252,35 @@ function canopy_boundary_fluxes!(
         t,
     )
     # Update the root flux of water per unit ground area in place
+    # Note that here we dispatch off of the ground conditions `bc.ground`
+    # to handle standalone canopy simulations vs integrated ones
```

Comment:
maybe we could just make this comment once and explain that it applies to the following 3 flux calculations

Reply:
will do!